xml-exemplo/Atendimento_Individual.esus.xml

```diff
@@ -98,17 +98,20 @@
 			<medicamentos>
 				<codigoCatmat>BR0387985</codigoCatmat>
 				<viaAdministracao>11</viaAdministracao>
-				<dose>1 aplicação</dose>
+				<dose>6 aplicações</dose>
 				<doseUnica>false</doseUnica>
 				<usoContinuo>false</usoContinuo>
-				<doseFrequenciaTipo>1</doseFrequenciaTipo>
+				<doseFrequenciaTipo>4</doseFrequenciaTipo>
 				<doseFrequencia>manhã</doseFrequencia>
 				<doseFrequenciaQuantidade>2</doseFrequenciaQuantidade>
 				<doseFrequenciaUnidadeMedida>3</doseFrequenciaUnidadeMedida>
 				<dtInicioTratamento>1586095200000</dtInicioTratamento>
 				<duracaoTratamento>12</duracaoTratamento>
 				<duracaoTratamentoMedida>3</duracaoTratamentoMedida>
 				<quantidadeReceitada>12</quantidadeReceitada>
+				<qtDoseManha>2</qtDoseManha>
+				<qtDoseTarde>2</qtDoseTarde>
+				<qtDoseNoite>2</qtDoseNoite>
```

Comment:
Só pensei em esses valores de dose serem distintos para quando essa ficha for utilizada como teste não mascare possíveis problemas